### PR TITLE
Wills Registry - Submit Request for Access#6872

### PR DESCRIPTION
**Pull request by shabeeb-aot**
*Issue #:*
https://github.com/bcgov/entity/issues/6872

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: auth-web/src/components/auth/account-settings/product/ProductPackage.vue [hunk 1]
```diff
@@ -0,0 +1,144 @@
+<template>
+  <v-container class="view-container">
+    <div class="view-header flex-column mb-6">
+      <h2 class="view-header__title" data-test="account-settings-title">
+        Product Packges
+      </h2>
+      <p class="mt-3 payment-page-sub">
+        Select multiple additional products this account require access. By default the account will have
+        access to adipiscing elit. Aliquam at porttitor sem.  Aliquam erat volutpat. Donec placerat.
```
Comment (saravanpa-aot) [inline auth-web/src/components/auth/account-settings/product/ProductPackage.vue:9]: can you please ask Roy for the correct wordings?

File: auth-web/src/components/auth/common/SingleProduct.vue [hunk 2]
```diff
@@ -0,0 +1,236 @@
+<template>
```
Comment (sumesh-aot) [inline auth-web/src/components/auth/common/SingleProduct.vue:1]: May be name it as Product, with no Single

File: auth-web/src/components/auth/common/SingleProduct.vue [hunk 3]
```diff
@@ -0,0 +1,236 @@
...
+  <div>
+    <template>
+      <v-card
+        outlined
+        hover
+        class="product-card py-8 px-5 mb-4 elevation-1"
+        :class="[{'px-8': icon === ''}, {'processing-card' : !isRRequesting}]"
```
Comment (sumesh-aot) [inline auth-web/src/components/auth/common/SingleProduct.vue:8]: isRRequesting correct? 2 Rs

**Review by saravanpa-aot — APPROVED** [review]

File: auth-web/src/components/auth/common/SingleProduct.vue [hunk 4]
```diff
@@ -0,0 +1,236 @@
...
+              <span>{{label}}</span>
+            </v-btn>
+          </header>
+
+          <div class="product-card-contents">
+            <v-expand-transition>
+              <div v-if="isRequestNow">
+                <div class="pt-7 mb-7" >
+                  <v-divider class="mb-7"></v-divider>
+                  <p class="mb-7">The search and registration products are intended for the exclusive use of solicitors and notaries only.
```
Comment (sumesh-aot) [inline auth-web/src/components/auth/common/SingleProduct.vue:44]: Better to move this to a component, as all product selection  won't be having same terms or fields

File: auth-web/src/components/auth/common/SingleProduct.vue [hunk 5]
```diff
@@ -0,0 +1,236 @@
...
+.pad-form-container {
+  max-width: 75ch;
+}
+.terms-error{
+  color: #d3272c;
+  font-size: 16px;
+  font-weight: bold;
+}
+.error-color{
+  color: #d3272c;
```
Comment (karthik-aot) [inline auth-web/src/components/auth/common/SingleProduct.vue:234]: can we use var(--v-error-base)

File: auth-web/src/routes/router.ts [hunk 6]
```diff
@@ -187,6 +188,11 @@ export function getRoutes (): RouteConfig[] {
           meta: {
             isPremiumOnly: true
           }
+        },
+        {
+          path: 'product-package',
```
Comment (sumesh-aot) [inline auth-web/src/routes/router.ts:193]: I think we are using product-settings as the route in settings endpoint, can you change route to product-settings

**Review by karthik-aot — APPROVED** [review]

**Pull request merged**